app/views/user_mailer/registration_confirmation.html.erb

```diff
@@ -0,0 +1,18 @@
+
+
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
+  </head>
+
+   <body>
+    <h1>Hi, <%= @user.full_name %>!!!</h1>
+    <p>
+      Thanks for registering! To confirm your registration click the URL below.
+      <%# confirm_email_user_url(@user.confirm_token) %>
+      <%= "http://localhost:3000/api/v1/confirmations?confirm_token=#{@user.confirm_token}&id=#{@user.id}"%>
```

Comment:
It's not ok))

Reply:
```
<%= "#{host}/api/v1/confirmations?confirm_token=#{@user.confirm_token}&id=#{@user.id}"%>

def host
  Rails.env.production? ? 'heroku link' : 'http://localhost:3000'
end
```

Reply:
Ok) should I make a prefix for the path to the endpoint (something like it `new_user_session_path`) or something else?

Reply:
idk
I think this url is ok
just adjust it according to the enviroment